### PR TITLE
docs: use mermaid for class diagram

### LINKED ISSUE #1182: [📑 Docs]: Use code to represent the internal models
### What Dev Docs changes are you proposing?

Currently, it's not easy to update the [internal model images](https://github.com/asyncapi/modelina/blob/master/docs/internal-model.md) and keeping them up to date, as I have the draw.io file...

Instead, we should add them as code through something like Mermaid: https://mermaid.js.org/syntax/classDiagram.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Qypol342**
**Description**
Corrected sentence:
- Update documentation internal-model class diagram from PNG to [Mermaid](https://mermaid.js.org/syntax/classDiagram.html#styling) allowing for easier updates. Preview result [here](https://github.com/Qypol342/modelina/blob/master/docs/internal-model.md)

**Related issue(s)**
Resolves #1182

### CODE REVIEW

File: docs/internal-model.md [hunk 1]
```diff
@@ -19,9 +19,125 @@ These are the meta models and their meaning:
...
+    }
+
+    class TupleModel{
+        <<class>>
+        TupleValueModel[] tupleModels
+    }
+
+    class ObjectModel{
+        <<class>>
+        String, ObjectPropertyModel properties
```
Comment (jonaslagoni) [inline docs/internal-model.md:48]: Looks like there are a few of the properties that are missing their types. In the old image, it's a dictionary with `string` key and `ObjectPropertyModel` value 🙂 

There are a few places this applies 🙂

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Nice @Qypol342, this is really needed 👍

File: docs/internal-model.md (hunk 1 above)
Comment (Qypol342) [reply docs/internal-model.md]: @jonaslagoni There you go! I've fixed it up. 🙃

**Review by jonaslagoni — COMMENTED** [review]
image

The `property --> MetaModel` is that correct? or is it being displayed incorrectly? 🤔

**Comment by Qypol342** [comment]
> image
> 
> The `property --> MetaModel` is that correct? or is it being displayed incorrectly? 🤔

I've just followed the **original** doc:
image

`property --> MetaModel` the way it is **written** is the mermaid **standard** to signify an association **without** the arrow **going** all over the schema

I can make it look the old **good** way by not **specifying** which **attribute** the association **comes** from.

**Review by jonaslagoni — APPROVED** [review]
Alright, let's keep it 👍

**Comment by jonaslagoni** [comment]
@all-contributors please add @Qypol342 for docs

**Comment by kennethaasan** [comment]
Really great stuff @Qypol342!

**Pull request merged**